doc/lightning-listpays.7.md

```diff
@@ -4,47 +4,51 @@ lightning-listpays -- Command for querying payment status
 SYNOPSIS
 --------
 
-**listpays** \[bolt11\]
+**listpays** \[bolt11\] \[payment_hash\]
 
 DESCRIPTION
 -----------
 
 The **listpay** RPC command gets the status of all *pay* commands, or a
-single one if *bolt11* is specified.
+single one if *bolt11* or *payment_hash* are specified (no both).
 
 RETURN VALUE
 ------------
 
 On success, an array of objects is returned. Each object contains:
 
- *bolt11*
+- *bolt11*: (if the payment was did with pay command)
 the *bolt11* argument given to *pay* (see below for exceptions).
 
- *status*
+- *payment_hash*: (is the payment was did with the keysend command)
+the *payment_hash* argument givent to *keysend*
+
+- *status*:
 one of *complete*, *failed* or *pending*.
 
- *payment\_preimage*
+- *payment\_preimage*:
 (if *status* is *complete*) proves payment was received.
 
- *label*
+- *label*:
 optional *label*, if provided to *pay*.
 
- *amount\_sent\_msat*
+- *amount\_sent\_msat*:
 total amount sent, in "NNNmsat" format.
 
-For old payments (pre-0.7) we didn’t save the *bolt11* string, so in its
-place are three other fields:
+~For old payments (pre-0.7) we didn’t save the *bolt11* string, so in its
```

Comment:
Please don't use strikethrough in the documentation, it should always reflect the latest state, and not carry old / outdated information at all. Also I'm pretty sure we should keep these warnings unchanged for now, until we really remove them.

Reply:
I will apport this change inside the next commit. Thanks for this review